_sass/frogtek-theme.scss

```diff
@@ -377,6 +377,39 @@ footer {
   }
 }
 
+.analytics-carousel {
+  .owl-dots {
+    color: $color-frogtek-orange;
```

Comment:
Including the name of the colour as part of the name of the variable is usually a bad choice (nowadays the semantics point of view is key)